include/openssl/ssl.h

```diff
@@ -2023,8 +2031,7 @@ void SSL_CTX_set_not_resumable_session_callback(SSL_CTX *ctx,
 
 void SSL_set_not_resumable_session_callback(SSL *ssl,
                                             int (*cb) (SSL *ssl,
-                                                       int
-                                                       is_forward_secure));
+                                                       int is_forward_secure));
```

Comment:
Is this going from column 79 to column 83?!

Reply:
Not on my side.